test/oauth_proxy_outbound_test.js

```diff
@@ -21,7 +21,7 @@ describe('oauth_proxy outbound message integrity: verbs', function() {
     ['GET', 'DELETE'].forEach(function(verb) {
```

Comment:
The latter 4 test templates are the ones that still fail in the build, even though they pass on [master](https://travis-ci.org/Cimpress-MCP/oauth_reverse_proxy/builds/81434564).

```bash
$ set OAUTH_REVERSE_PROXY_LOG_LEVEL="trace"; mocha --reporter spec -t 10000 ./test/oauth_proxy_outbound_test.js | bunyan

  64 passing (494ms)
  8 failing
```

@ryanbreen any idea why? I've tried rolling back my changes one by one locally, and looking through the diff in this PR, but I can't pinpoint the line(s) that cause these tests to fail.